src/main/java/net/minecraftforge/fml/client/gui/screen/ModListScreen.java

```diff
@@ -123,7 +123,7 @@ public ModListScreen(Screen mainMenu)
     {
         super(new TranslationTextComponent("fml.menu.mods.title"));
         this.mainMenu = mainMenu;
-        this.mods = Collections.unmodifiableList(ModList.get().getMods());
+        this.mods = Collections.unmodifiableList(net.minecraftforge.fml.ModList.get().getMods());
```

Comment:
Couldn’t this be an import instead of a fully qualified reference?

Reply:
https://github.com/MinecraftForge/MinecraftForge/wiki/If-you-want-to-contribute-to-Forge#conventions-for-coding-patches-for-a-minecraft-class-javapatch

Reply:
This isn't a patch file.